collectors/proc.plugin/sys_fs_btrfs.c

```diff
@@ -205,13 +189,8 @@ static inline int find_btrfs_disks(BTRFS_NODE *node, const char *path) {
             continue;
         }
 
-        if(read_single_number_file(d->hw_sector_size_filename, &d->hw_sector_size) != 0) {
-            collector_error("BTRFS: failed to read '%s'", d->hw_sector_size_filename);
-            d->exists = 0;
-            continue;
-        }
-
-        node->all_disks_total += d->size * d->hw_sector_size;
+        // /sys/block/<name>/size is in fixed-size sectors of 512 bytes
```

Comment:
nitpick: probably add the [link shared](https://github.com/torvalds/linux/blob/ffe78bbd512166e0ef1cc4858010b128c510ed7d/include/linux/types.h#L120-L121) by @Dim-P

Reply:
+1